clang/lib/Basic/Targets/RISCV.h

```diff
@@ -134,6 +134,25 @@ class RISCVTargetInfo : public TargetInfo {
 
   bool validateGlobalRegisterVariable(StringRef RegName, unsigned RegSize,
                                       bool &HasSizeMismatch) const override;
+
+  bool checkCFProtectionBranchSupported() const override {
+    return ISAInfo->hasExtension("zicfilp");
```

Comment:
Branch protection doesn't really require zicfilp. The encoding of lpad is a valid encoding on all RISC-V CPUs. That's why x86 only checks that the CPU is at least a PentiumPro.